Tasks/Common/packaging-common/locationUtilities.ts

```diff
@@ -37,7 +37,7 @@ export async function getServiceUriFromAreaId(serviceUri: string, accessToken: s
 
     tl.debug(`Getting URI for area ID ${areaId} from ${serviceUri}`);
     try {
-        const serviceUriFromArea = await locationApi.getResourceArea(areaId);
+        const serviceUriFromArea = await retryOnExceptionHelper(() => locationApi.getResourceArea(areaId), 5, 1000);
```

Comment:
is this code used by the nuget authenticate task?

Reply:
or is there a separate place to update for that?

Reply:
Not the new one, I am first going to first verify this and then port the code over there.